raw-spans-grouper/raw-spans-grouper/src/main/resources/configs/common/application.conf

```diff
@@ -41,6 +41,11 @@ kafka.streams.config = {
 span.groupby.session.window.interval = 30
 span.groupby.session.window.interval = ${?SPAN_GROUPBY_SESSION_WINDOW_INTERVAL}
 
+span.window.store.retention.time.mins = 60
+span.window.store.retention.time.mins = ${?SPAN_WINDOW_STORE_RETENTION_TIME_MINS}
```

Comment:
are these also defined in helm templates ?

Reply:
added